formvalidation.js

```diff
@@ -0,0 +1,24 @@
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+
+form.addEventListener('submit', (e) => {
+  let valid = true;
+  const email = document.getElementById('email').value;
+  const emailArray = email.split('');
+
+  for (let i = 0; i < emailArray.length; i += 1) {
+    if (
+      emailArray[i] !== emailArray[i].toLowerCase()
+            && emailArray[i] !== '@' && emailArray[i] !== '.'
+    ) {
+      valid = false;
+    }
+  }
+
+  if (valid) {
+    error.innerHTML = '';
+  } else {
+    e.preventDefault();
+    error.innerHTML = 'Please enter your email in lower case only';
+  }
+});
```

Comment:
- [ ] Great job so far but your form validation does not work. Rather than looping through each letter of the email, you can simply use the comparison `email === email.toLowerCase()` and if this condition is not valid you can then prevent the default form submission and show your error message. Kindly fix this so as to improve the user interface  👍

Reply:
- [ ] Good job trying to implement the changes but the validation but it still does not work. As suggested by the previous reviewer, simply just compare the value from the email input with its lowercased value using `email === email.toLowerCase()`